pkg/rsyslog/rsyslog.conf

```diff
@@ -0,0 +1,81 @@
+# rsyslog v5: load input modules
```

Comment:
What is the purpose of this conf file for now?

Is there any useful function it is trying to perform (in addition of simply making sure that rsyslogd comes up)?

Reply:
This file is mostly to make rsyslog come up. It has some transformations (templates) that changes non-json messages to json format and also inject the partition name, process name json fields.